pkg/scheduler/apis/config/validation/validation.go

```diff
@@ -17,12 +17,25 @@ limitations under the License.
 package validation
 
 import (
+	"errors"
+	"fmt"
+
+	v1 "k8s.io/api/core/v1"
+	utilerrors "k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/apimachinery/pkg/util/validation"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 	componentbasevalidation "k8s.io/component-base/config/validation"
+	v1helper "k8s.io/kubernetes/pkg/apis/core/v1/helper"
 	"k8s.io/kubernetes/pkg/scheduler/apis/config"
 )
 
+// The following strings are duplicated here to avoid introducing dependency to pkg/scheduler/framework/...
+const (
+	// MaxNodeScore is the maximum score a Score plugin is expected to return.
+	MaxNodeScore int64 = 10
+)
+
```

Comment:
I don't understand why are we defining this here again.

Reply:
They are removed.